app/services/InputCSVParser.java

```diff
@@ -175,6 +175,7 @@ public void parseCSV(EntityManager em, DimensionalDataSet dds, File inputFile) {
             try {
                 csvReader.readLine();
                 while (csvReader.ready() && (rowData = csvParser.parseLine(csvReader.readLine())) != null) {
+                    firstCellVal = rowData[0];
```

Comment:
Actually no, we can't as the code is currently because it lets the value of 'fitstCellVal' sort of leak over into a wider scope to use in the only place where it is actively used.